bigquery.tf

```diff
@@ -3,6 +3,50 @@ locals {
   bigquery_dataset_name         = coalesce(var.bigquery_dataset_name, local.default_bigquery_dataset_name)
 }
 
+resource "google_bigquery_connection" "bq_spark_connection" {
+  connection_id = "bq-spark-conn-${lower(var.organisation_id)}-${lower(var.country_code)}"
+  location      = var.storage_location
+  description   = "BQ spark connection for ML"
+  spark {
+    spark_history_server_config {
+      dataproc_cluster = google_dataproc_cluster.dataproc_persistent_history_server.id
+    }
+  }
+}
+
+resource "google_dataproc_cluster" "dataproc_persistent_history_server" {
```

Comment:
as discussed in the channel, the cluster block will be removed
https://liveramp.slack.com/archives/C03D27C80HM/p1722441087320019?thread_ts=1722420289.782479&cid=C03D27C80HM

Reply:
Done